webpack.config.js

```diff
@@ -1,20 +1,29 @@
-const path    = require("path")
-const webpack = require("webpack")
+const path = require('path');
+const webpack = require('webpack');
 
 module.exports = {
-  mode: "production",
-  devtool: "source-map",
+  mode: 'production',
+  devtool: 'source-map',
   entry: {
-    application: "./app/javascript/application.js"
+    application: './app/javascript/application.js',
```

Comment:
- [ ] Please check your webpack configuration, at the moment, I'm having an error when I try to run `./bin/dev`:

![image](https://user-images.githubusercontent.com/77069011/224429220-844732e4-0ec5-48a9-839d-b158ed67d30c.png)

This is because here you specified as entry point `'./app/javascript/application.js'` but this file doesn't exist, please consider fixing this! 🤓 🔧